docs/standard/library-guidance/strong-naming.md

```diff
@@ -39,7 +39,9 @@ The benefits of strong naming are:
 You should strong name your open-source .NET libraries. Strong naming an assembly ensures the most people can use it, and strict assembly loading only affects the .NET Framework.
 
 > [!NOTE]
```

Comment:
perhaps the NOTE should be changed to IMPORTANT for added emphasis?

Reply:
I think note is ok. It isn't a big problem if someone wants to SN their own application libraries.

Also there is another !IMPORTANT just below this that is more... important

Reply:
I agree, @JamesNK; I'd missed the IMPORTANT just below. This looks good, so I'll merge your PR now.